tasks_week3_omer_gertler1/include/Animal.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+//#include <string>
+//#include <ostream>
+#include "Location.h"
+
+using std::string;
+//using std::ostream;
+
+class Animal
+{
+protected:
+
+private:
+	string name = "";
+	string type = "";
+	Location loc;
+	bool isMoving = true;
+
+public:
+	Animal();
+	Animal(const std::string& name, const Location& loc);
+
```

Comment:
no use of virtual destructor 
check this
https://stackoverflow.com/questions/461203/when-to-use-virtual-destructors